xml/admin_ceph_gateway.xml

```diff
@@ -3001,24 +3001,6 @@ rgw_override_bucket_index_max_shards = 12
 &prompt.cephuser;ceph osd pool rm default.rgw.gc default.rgw.gc --yes-i-really-really-mean-it
 &prompt.cephuser;ceph osd pool rm default.rgw.log default.rgw.log --yes-i-really-really-mean-it
 &prompt.cephuser;ceph osd pool rm default.rgw.users.uid default.rgw.users.uid --yes-i-really-really-mean-it</screen>
-   </sect3>
-   <sect3 xml:id="ceph-rgw-secondzone-update-config">
-    <title>Update the &ceph; Configuration File</title>
-    <para>
-     Update the &ceph; configuration file on the secondary zone hosts by adding
-     the <literal>rgw_zone</literal> configuration option and the name of the
-     secondary zone to the instance entry.
-    </para>
-<screen>[client.rgw.<replaceable>INSTANCE-NAME</replaceable>]
-...
-rgw_zone=<replaceable>ZONE-NAME</replaceable></screen>
-    <para>
-     For example:
-    </para>
-<screen>[client.rgw.rgw2]
-host = rgw2
-rgw frontends = "civetweb port=80"
-rgw_zone=us-west</screen>
```

Comment:
just replace this conf section with `ceph config set <service-name> rgw_zone us-west `...